dali/ft/filecopy.cpp

```diff
@@ -830,7 +840,12 @@ void FileSprayer::beforeTransfer()
                     splitUNCFilename(remoteFilename.str(), &remoteDirectory, &remoteDirectory, NULL, NULL);
 
                     Owned<IFile> dir = createIFile(remoteDirectory.str());
+                    bool modPerms = false;
+                    if (!dir->exists() && (umask != -1))
+                        modPerms = true;
```

Comment:
Not new, but if dir already exists, it shouldn't not be [trying to] create it either.